koji_containerbuild/plugins/cli_containerbuild.py

```diff
@@ -105,6 +107,8 @@ def parse_arguments(options, args, flatpak):
     parser.add_option("--skip-build", action="store_true",
                       help=_("Skip build and update buildconfig. "
                              "Use this option to update autorebuild settings"))
+    parser.add_option("--userdata",
+                      help=_("Dictionary of user defined custom metadata"))
```

Comment:
Let's say it's a JSON dictionary, so people know not to use single-quotes or something.

Reply:
Fixed, thanks.